ios/reactNativeApp.xcodeproj/project.pbxproj

```diff
@@ -5,7 +5,6 @@
 	};
 	objectVersion = 46;
 	objects = {
-
```

Comment:
Just out of curiosity, why do these changes happen from time to time?

Reply:
Yeah I don't really know, it looks it differs depending who makes them... I guess we can investigate it in the future but yeah...